GoogleForCreators/web-stories-wp
# Autosave: trigger autosave regardless of being idle

## Context



## Summary
Removes autosave deps from `story` and `pages` which resulted in autosave being triggered only when idle for the time of the interval.


## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1. Create a story
2. Continue making edits for 60 seconds
3. Verify the autosave triggers without having to be idle for 60 seconds
4. Try once more (without refreshing) to ensure it triggers more than once.


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #12809
